BackEnd/src/main/java/com/exadel/carpoolfree/controller/UserController.java

```diff
@@ -0,0 +1,51 @@
+package com.exadel.carpoolfree.controller;
+
+import com.exadel.carpoolfree.model.User;
+import com.exadel.carpoolfree.repository.UserRepository;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
```

Comment:
I think we need all our APIs ti start with `/api/*` so this one should read as `/api/users`

Reply:
anything else looks just fine

Reply:
fixed